pages/map.js

```diff
@@ -0,0 +1,28 @@
+import Map from "../components/Map";
+import Heading from "../components/Heading";
+import { StyledMain } from "../components/styles";
+import useSWR from "swr";
+
+export default function MapPage({ username, onLogin }) {
+  const { data: restaurants } = useSWR("/api/restaurants", {
+    fallbackData: [],
+  });
+  const { data: reserves } = useSWR("/api/reserves", {
+    fallbackData: [],
+  });
+  const dataForMapMarkers = restaurants.filter((restaurant) =>
+    reserves.some((reserve) => reserve.restaurantId === restaurant._id)
+  );
+
+  return (
+    <>
```

Comment:
```suggestion

```

Reply:
thanks a lot Augustin. Your suggestions are useful for me. but here I can only keep it, because there are 2 elements in this page.